ui/preview/model.yml

```diff
@@ -104,6 +104,16 @@ site:
         version: '0.0.1'
         displayVersion: '0.0.1'
       latestVersion: *latest_version_community_contracts
+    stellar-contracts:
```

Comment:
If you click Stellar Contracts on the preview, it links to https://deploy-preview-427--openzeppelin-docs.netlify.app/#stellar-contracts::index.adoc which doesn't work.

I'm not sure why, but perhaps it relates to the version identifier in the branch https://github.com/OpenZeppelin/stellar-contracts/blob/release-v1.0.0/docs/antora.yml being `0.1.0`.

Reply:
I believe this problem is fixed 👍 
I can navigate to our documents now! 🚀